scripts/openwebnet-discovery.nse

```diff
@@ -137,10 +137,15 @@ local function get_response(sd, request)
 
     if status == nil then
       stdnse.debug("Error: " .. tostring(data))
-      -- Captures the NACK after TIMEOUT occured.
-      -- Avoids false results.
-      status, data = sd:receive_buf("##", true)
-      break
+      if tostring(data) == "TIMEOUT" then
```

Comment:
I thought if we get some new kind of error (I'm not sure what the new kind of error will be), converting that into `tostring` might help us out while printing it.